data/libs/concoctions_lib.lua

```diff
@@ -0,0 +1,164 @@
+local updateInterval = 60
+
+-- Concoctions registry
+local concoctions = {}
+
+ConcoctionTickType = {
+	Online = "online",
+	Experience = "experience",
+}
+
+-- Concoction class
+Concoction = {}
+Concoction.__index = Concoction
+Concoction.Ids = {
+	KooldownAid = 36723,
+	StaminaExtension = 36725,
+	StrikeEnhancement = 36724,
+	CharmUpgrade = 36726,
+	WealthDuplex = 36727,
+	BestiaryBetterment = 36728,
+	EarthResilience = 36729,
+	FireResilience = 36730,
+	IceResilience = 36731,
+	EarthAmplification = 36732,
+	EnergyResilience = 36733,
+	DeathResilience = 36734,
+	HolyResilience = 36735,
```

Comment:
```suggestion
	HolyResilience = 36735,
```
O cliente está crashando com esse id quando aperta o botão 'Bosstiary' no cyclopedia.|

Consegui usar assim, porém apenas dois potions ficam desativados.

```Concoction.Ids = {
	KooldownAid = 36723,
	StaminaExtension = 36725,
	StrikeEnhancement = 36724,
	CharmUpgrade = 36726,
	WealthDuplex = 36727,
	BestiaryBetterment = 36728,
	FireResilience = 36729,
	EarthResilience = 36731,
	IceResilience = 36730,
	EnergyResilience = 36732,
	DeathResilience = 36734,
	PhysicalResilience = 36735,
	FireAmplification = 36736,
	IceAmplification = 36737,
	EarthAmplification = 36738,
	EnergyAmplification = 36739,
	HolyAmplification = 36740,
	DeathAmplification = 36741,
}
```

Reply:
realmente os ids tao errado, vou arrumar, mas nao entendi o crash

Reply:
> realmente os ids tao errado, vou arrumar, mas nao entendi o crash

Opa, sim, ele tem um conflito há algum tempo.

Reply:
explica melhor isso ai, com que vc causou o crash?

Reply:
Pode ser um ID incorreto que cause conflitos relacionados a esse problema.